### ISSUE TITLE
Uploaded files given full Windows path as filename

### ISSUE THREAD

**Issue by rdagger** [jczic/ESP32-MPY-Jama#23]
I uploaded a few files from a Windows 10 PC to an ESP32 Lolin32 Pro.  The files were renamed to the full Windows source path.
For example:

After upload -->
```
MicroPython >>> os.listdir()
['C:\\Users\\Registered User\\Desktop\\test\\ads8671.py', 'C:\\Users\\Registered User\\Desktop\\test\\ftp.py', 'boot.py']

```
This is what the filenames should be -->
```
MicroPython >>> os.listdir()
['ads8671.py', 'boot.py', 'ftp.py']
```

Also, it would be great if you could select multiple files to upload.  Thanks!

**Comment by jczic** [comment]
I will check that!

**Comment by jczic** [comment]
@rdagger, it's fixed : 44e151a1dedf7b02d54c48091eb3a9e7a5626d95
